### DISCUSSION TITLE
When using RVD, how can I split the input surface mesh edges and the clipped bisectors in the final output mesh?

### DISCUSSION THREAD

**Discussion by fourdimension** [BrunoLevy/geogram#177] [Q&A] [answered]
It seems that when I use compute_RVD_cells or RVD->compute_RVD(M_out), the surface mesh edges will merge to the final output mesh. But can I only get the intersectrion between the bisectors and the intersection between the bisector and the surface(excluding the original mesh edges)? 

I found the same questions in another mirror repo, but cannot find the answer online. The link is attached below.

https://github.com/alicevision/geogram/issues/1

**Comment by BrunoLevy** [comment, marked as answer]
See answers to [this question](https://github.com/BrunoLevy/geogram/discussions/100)
